build_scripts/dependencies_static_analysis_test_cov.py

```diff
@@ -51,9 +51,8 @@ def check_md_linting():
     """
     function check the repo for any yml linting errors
     """
-    print("----TODO----- Markdown linting")
-    # call_subprocess("markdownlint *.md ")
-    # print("Stage linting md files -- COMPLETED & PASSED  --")
+    call_subprocess("markdownlint *.md ")
```

Comment:
Nice touch - good to do linting for markdown. 

Can you also consider doing spell-checks for markdowns?

Reply:
Spell check for markdown is done through the current Aspell. in place.